packages/lodestar/src/eth1/impl/interop.ts

```diff
@@ -0,0 +1,44 @@
+import {EventEmitter} from "events";
+
+import {bytes32, number64} from "@chainsafe/eth2.0-types";
+
+import {IEth1Notifier} from "../";
+import {Block} from "ethers/providers";
+
+export class InteropEth1Notifier extends EventEmitter implements IEth1Notifier {
```

Comment:
Rename to `quickStartEth1Notifier`? Interop is a fairly specific name, people reading the code base may get confused.

Reply:
It's for interop, eventually it will folow interop mock eth1 document. 
https://github.com/ethereum/eth2.0-pm/tree/master/interop/mocked_eth1data

Reply:
hmmm i guess ya..